[travis] Encrypt access key ID in travis yml file

Zyiqin-Miranda opened the pull request:
<!-- Thank you for your contribution! Please review https://github.com/amzn/amazon-ray/blob/master/CONTRIBUTING.md before opening a pull request. -->

<!-- Please add a reviewer to the assignee section when you create a PR. If you don't have the access to it, we will shortly find a reviewer and assign them to your PR. -->

## Why are these changes needed?

This PR adds encryption to access key ID in travis yml file.

## Related issue number

<!-- For example: "Closes #1234" -->

## Checks

- [x] I'm working against the latest source on the **experimental** branch.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

pdames commented:
Thanks! The changes look good but, before merging , can you rebase on top of the latest commit from the experimental branch, and change the merge target branch for this PR from main to experimental?

Zyiqin-Miranda commented:
> Thanks! The changes look good but, before merging , can you rebase on top of the latest commit from the experimental branch, and change the merge target branch for this PR from main to experimental?

Thanks Patrick, changed target branch and rebased on top of latest commit
